internal/pkg/addon/addons.go

```diff
@@ -49,13 +49,7 @@ type Stack struct {
 // files found there. If no addons are found, Parse returns a nil
 // Stack and ErrAddonsNotFound.
 func Parse(workloadName string, ws workspaceReader) (*Stack, error) {
-	path, err := ws.WorkloadAddonsPath(workloadName)
-	if err != nil {
-		return nil, &ErrAddonsNotFound{
-			WlName:    workloadName,
-			ParentErr: err,
-		}
-	}
+	path := ws.WorkloadAddonsPath(workloadName)
```

Comment:
nit: now that we defined the error out of existence 💅 should we just inline these temp variables?
```suggestion
	ws.ListFiles(ws.WorkloadAddonsPath(workloadName))
```